pkg/asset/cluster/tfvars.go

```diff
@@ -9,12 +9,6 @@ import (
 	igntypes "github.com/coreos/ignition/config/v2_2/types"
 	gcpprovider "github.com/openshift/cluster-api-provider-gcp/pkg/apis/gcpprovider/v1beta1"
 	libvirtprovider "github.com/openshift/cluster-api-provider-libvirt/pkg/apis/libvirtproviderconfig/v1beta1"
-	"github.com/pkg/errors"
```

Comment:
we like to keep the imports groups in internal(stdlib), external, installer

Reply:
These were moved due to ide formatting (gofmt, I think). I moved them back. I think it was trying to move them based on alphabetical order.

Reply:
actually this is causing `gofmt` test to fail: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_installer/2406/pull-ci-openshift-installer-master-gofmt/5863

Is there a way around that? (extra line perhaps)?

Reply:
there was a newline separating each group.